NYPL/cantaloupe
# DR-3034 jruby base image and rsolr gem

* Switches the base Docker image from (deprecated) openjdk to jruby
* Enables gem installation in Dockerfile
* Installs rsolr gem and requires it in delegates.rb to show it is successfully installed

How to test:
* docker-compose build
* docker-compose up
* load an image (I use http://localhost:8182/iiif/2/53926/full/full/0/default.jpg)
* It should work without complaining about gems not being installed!
